strings/src/main/res/values-mr/strings.xml

```diff
@@ -149,8 +149,7 @@
   <string name="choose_image">प्रतिमा निवडा</string>
   <string name="selected_invalid_image">निवडलेली फाईल वैध प्रतिमा नाही</string>
   <string name="take_picture_instruction">एक चित्र घेण्यासाठी स्क्रीन टॅप करा</string>
-  <string name="camera_error">हे साधन कॅमेरॅ2 API ला समर्थन देत नाही.</string>
-  <!--Dialog heading for saving annotated image-->
+    <!--Dialog heading for saving annotated image-->
```

Comment:
It looks like this is a problem with the Transifex files using different indentation from Android Studio('s standard settings) so when I deleted this string via the editor I got the "correct" 4 space indent rather than Transifex's 2 spaces. Rather than go through and fix these all individually (or correctly format all the translated string files), I reckon we just leave it, and it'll fix itself when the new Transifex files are generated I imagine. 

@lognaturel is there a way we can change the indent in Transifex so that we can avoid this problem in future?

Reply:
I don't see a way to do it automatically as part of the Transifex export. I could do a post-processing step but I'd rather keep that pipeline as simple as possible.

When you say "the editor", you mean the translation editor? What if you use the Refactor tool? My preference would be to just not worry about this.